data/formats-data.js

```diff
@@ -6480,6 +6510,22 @@ exports.BattleFormatsData = {
 		eventOnly: true,
 		tier: "Uber",
 	},
+	poipole: {
+		tier: "NFE",
```

Comment:
Does poipole need to be event only? I don't believe it is supposed to be useable in Little Cup.

Reply:
I'm waiting on adding the eventOnly because i dont know the event yet; itll be updated as soon as it's all confirmed

Reply:
It is a gift obtained in game with level 40, so it is not useable in Little Cup. Most of those walkthrough videos on youtube can confirm this.

Reply:
Yep we found the event set. All good.